index.js

```diff
@@ -24,71 +24,99 @@ module.exports = (opts = {}, ctx) => ({
       .action(async (dir = '.') => {
         dir = join(process.cwd(), dir)
         try {
-          const nCtx = await dev({
+          const devContext = await dev({
             sourceDir: dir,
             clearScreen: false,
-            theme: opts.theme || '@vuepress/default'
+            theme: options.theme || '@vuepress/default'
```

Comment:
I just noticed this pre-existing this `theme: options.theme` line doesn't work well with your change to use an array for `options`.  In my own copy I'm going to be pushing your array down as an attribute within options.